### DISCUSSION TITLE
How to use uc_switch_to_frame()

### DISCUSSION THREAD

**Discussion by MacMarde** [seleniumbase/SeleniumBase#2358] [Q&A] [answered]
@mdmintz 
From the SB example [here](https://github.com/seleniumbase/SeleniumBase/blob/570910c6f6e079cddd00c23e826e202c6489f39e/examples/raw_turnstile.py#L11), I am wondering how to switch back from iframe to default content without being detected?

This is my code:
```
browser.uc_switch_to_frame("iframe")
browser.uc_click("span.mark")
browser.assert_element("img#captcha-success", timeout=15)            
browser.switch_to.default_content()
```

Is it save to use `browser.switch_to.default_content()` in terms of being undetected?

**Comment by mdmintz** [comment, marked as answer]
Using `driver.reconnect(0.15)` will automatically switch out of any iframes.
It's the equivalent of `driver.switch_to.default_content()` in UC Mode, but with the disconnect action to prevent detection.

**Comment by MacMarde** [comment]
Nice. Thank you.
